### DISCUSSION TITLE
İbax.io airdop beklentinizi sizce karşılayacak mı?

### DISCUSSION THREAD

**Discussion by Dedeko55** [IBAX-io/go-ibax#1441] [Soru Cevap - Türkçe]
İbax.io beklentileri karşılayacak kadar büyük bir proje mi 
Sizce?

**Comment by ZagorKahraman** [comment]
> İbax.io beklentileri karşılayacak kadar büyük bir proje mi 
> Sizce?

Today, data thefts cause an average of 4 million dollars in losses per attack in companies, so cyber security spending has increased significantly. However, although companies make huge investments in cyber security, company owners sometimes do not have a sufficient idea about the cyber attacks that their companies may encounter. Most business owners are people who have not spent the majority of their careers on the issue, at a time when cybercrime poses a serious threat to companies. So they often leave their cybersecurity strategies to their company's security experts or technology teams. However, company owners also need to be aware of their companies' potential cyber vulnerabilities, defense strategies, and reactions to a possible data security breach. For this reason, company owners should definitely ask the following questions in order to increase their cyber security knowledge and protect their companies.

**Reply by Yusufknk** [reply]
Bunu ilerleyen zamanlarda birlikte göreceğiz.

**Comment by Skips1** [comment]
Kesinlikle karşılamayacak.
